samples/common/src/misc.rs

```diff
@@ -0,0 +1,220 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
can we come up with a more descriptive filename than `misc.rs`? even if it's just utils?

Reply:
Changed.